dhcpv4/option_rfc1035label.go

```diff
@@ -0,0 +1,53 @@
+package dhcpv4
```

Comment:
isn't this the same as dhcpv6/option_rfc1035label.go ? Shall we extract it instead of duplicating?

Reply:
Yes, the same. Do you mean extract it to a directory inside `insomniacslk/dhcp`?
Sounds good. How should we call it?